Source/DafnyDriver/CliCompilation.cs

```diff
@@ -183,12 +183,12 @@ public class CliCompilation {
         if (Options.Get(CommonOptionBag.ProgressOption)) {
           var token = BoogieGenerator.ToDafnyToken(false, boogieUpdate.VerificationTask.Split.Token);
           var runResult = completed.Result;
-          var resourcesUsed = runResult.ResourceCount.ToString("E1", CultureInfo.InvariantCulture);
+          var timeString = runResult.RunTime.ToString("g");
           Options.OutputWriter.WriteLine(
             $"Verification part {canVerifyResult.CompletedParts.Count}/{canVerifyResult.Tasks.Count} of {boogieUpdate.CanVerify.FullDafnyName}" +
             $", on line {token.line}, " +
             $"{DescribeOutcome(Compilation.GetOutcome(runResult.Outcome))}" +
-            $", taking {runResult.RunTime.Milliseconds}ms and consuming {resourcesUsed} resources");
+            $" (time: {timeString}, resource count: {runResult.ResourceCount})");
```

Comment:
Why do you want to put the two time properties in parenthesis, instead of include them in the sentence? Also why "time" instead of "duration" ?

Reply:
Ultimately this makes the message shorter, making it more likely that it'll fit on a single line. When you're seeing many of these go by, the ability to quickly do pattern recognition on the output without explicitly reading it is very helpful, in my experience.

Reply:
Could you change "time" into "duration" ?

Reply:
Thinking about this more, I'm slightly inclined to make it more like the IDE and include something like `(0:00:1.25, 23947 RU)`. I'm worried about using multiple terms to describe resource usage. So far, we've always said "resource count" as an overall descriptor or used the abbreviation "RU" for the units, and the IDE uses "RU" heavily.

When using a unit suffix instead of a descriptive prefix, having either "time: " or "duration: " before the other one feels inconsistent to me. And it'll be formatted in such a way that I think it's obvious that it's describing a time/duration.

Reply:
One solution would be to list units for both, and not format the duration in "H:M:S" structure. So it would be something like "(3274.86 s, 23194090 RU)". I think perhaps that's my favorite, because it also makes high duration easier to pick out.